src/app/blog/multipath-will-fix-this/page.mdx

```diff
@@ -0,0 +1,316 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import {ThemeImage} from '@/components/ThemeImage'
+
+export const post = {
+  draft: false,
+  author: 'Floris Bruynooghe',
+  date: '2025-07-29',
+  title: 'Multipath Will Fix This',
+  description:
+    "A lightweight rpc crate for iroh protocols",
+}
+
+export const metadata = {
+    title: post.title,
+    description: post.description,
+    openGraph: {
+      title: post.title,
+      description: post.description,
+      images: [{
+        url: `/api/og?title=Blog&subtitle=${post.title}`,
+        width: 1200,
+        height: 630,
+        alt: post.title,
+        type: 'image/png',
+      }],
+      type: 'article'
+    }
+}
+
+export default (props) => <BlogPostLayout article={post} {...props} />
+
+
+Iroh is a library to establish direct peer-to-peer QUIC connections.
+This mean iroh does NAT traversal, colloquially known as holepunching.
```

Comment:
A link for what NAT traversal is (or maybe just a NAT), might be nifty.

Reply:
Two candidate links:
- https://en.wikipedia.org/wiki/NAT_traversal
- https://tailscale.com/blog/how-nat-traversal-works

I'm kind of leaning towards the tailscale link, if we are happy to link to them.

Reply:
I chose wikipedia, I think the tailscale link makes less sense here